src/pages/talks/2016-03-building-static-sites-with-react.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import withStyles from 'react-jss';
+
+import Talk from './_talk';
+
+const styles = theme => ({
+  root: {
+    width: '100%'
+  }
+});
+
+const BuildingStaticSites = ({ classes }) => (
+  <Talk>
+    {{
+      title: 'Building Static Sites with React',
+      subtitle: (
+        <div>
+          Ian Sinnott <span>from</span> TruSTAR Technologies
```

Comment:
I think it would be better to use links for Ian Sinnott and TruSTAR Technologies. Don't use the span. Simply leave it as it is and use the links for the other text.

https://www.iansinnott.com
https://www.trustar.co

The links should use `<a>` tag and `target="_blank"`.